src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/LeadershipJudgeFtpaDecisionHandler.java

```diff
@@ -123,6 +126,21 @@ public boolean canHandle(
         asylumCase.write(
             valueOf(String.format("IS_FTPA_%s_DECIDED", ftpaApplicantType.toUpperCase())), YES);
 
+        String currentDecision =
+            asylumCase.read(
+                valueOf(String.format("FTPA_%s_DECISION_OUTCOME_TYPE", ftpaApplicantType.toUpperCase())), String.class)
```

Comment:
We should keep full enum names even if there is a need to make some duplication in the code, script to validate release dependency on CCD defs is not that smart to parse String.format

Reply:
I think this entire class will need some refactoring as the `%s` is used throughout...

Reply:
I'll look to refactor this class (which was developed for ticket RIA-3211) as a separate exercise. Will leave in QA for the moment to allow testing to continue (based on urgency expressed by the DM).